lexicons/tools/ozone/set/removeSet.json

```diff
@@ -0,0 +1,23 @@
+{
+  "lexicon": 1,
+  "id": "tools.ozone.sets.removeSet",
+  "defs": {
+    "main": {
+      "type": "procedure",
+      "description": "Remove an entire set",
```

Comment:
I would also say in the description whether there is an error if there is an attempt to delete a set which doesn't exist.

@devinivy should we have a house style on this behavior (error or not)? IIRC deleteRecord doesn't error. maybe there is an HTTP success response code variant we could use to indicate this situation?

this isn't blocking feedback, we could move ahead without addressing this one.

Reply:
There's no success status code that I know of— I'd recommend adding a field to the response if you both want idempotent deletions (which I support) and knowledge of whether anything got deleted.

Reply:
Added the error case to the description and also added a specific error type for SetNotFound. 
Leading it with an empty response body, should be easy to add a success/failure flag later down the road.